src/network/networkd-address.c

```diff
@@ -953,8 +953,8 @@ int config_parse_lifetime(const char *unit,
                           void *data,
                           void *userdata) {
         Network *network = userdata;
-        _cleanup_(address_free_or_set_invalidp) Address *n = NULL;
```

Comment:
why drop this? the address should be invalidated unless the function succeeeds fully

Reply:
Ooops, indeed.